### PR TITLE
Tiny README patch

### PR DESCRIPTION
**Pull request by alecandido**
This is a PR just because we can not edit main :P

### CODE REVIEW

**Review by andrea-pasquale — APPROVED** [review]

**Comment by andrea-pasquale** [comment]
Also the documentation contains the same typo

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ Qibocal provides Quantum Characterization Validation and Verification protocols
 
 Qibocal key features:
 
-- Automatization of calibration protocols.
+- Automation of calibration protocols.
```
Comment (Edoardo-Pedicillo) [inline README.md:10]:
```suggestion
```
Since at the moment is not well defined

Comment (alecandido) [reply README.md:10]: This is also possible, @andrea-pasquale?

Comment (andrea-pasquale) [reply README.md:10]: Here it depends on what you mean by automation.
@Edoardo-Pedicillo is right, I think. After we merge #869 we could put something like
```suggestion
- Easy composition of calibration protocols
```
For the time being removing it is fine I think.

Comment (alecandido) [reply README.md]: Done in f1498e8

**Review by Edoardo-Pedicillo — APPROVED** [review]

**Pull request merged**